cli/fossilize_feature_filter.cpp

```diff
@@ -1000,6 +1003,11 @@ bool FeatureFilter::Impl::descriptor_set_layout_is_supported(const VkDescriptorS
 					mutable_counts.storage_image = 1;
 					break;
 
+				case VK_DESCRIPTOR_TYPE_SAMPLER:
+					// Sampler is a non-standard type for mutable
+					must_check_set_layout_before_accept = true;
```

Comment:
Should also set the mutable counts I think. It just also needs to set this bool to true.

Reply:
Should be fixed.